frontend/src/queries/utils.ts

```diff
@@ -272,3 +274,48 @@ export function isHogQlAggregation(hogQl: string): boolean {
         hogQl.includes('max(')
     )
 }
+
+export interface HogQLIdentifier {
+    __hogql_identifier: true
+    identifier: string
+}
+
+function hogQlIdentifier(identifier: string): HogQLIdentifier {
```

Comment:
Should we stick to the "uppercase acronyms" convention also in camelcase?

```suggestion
function hogQLIdentifier(identifier: string): HogQLIdentifier {
```

Reply:
IMO in camel case acronyms should be capitalized like words. E.g. `xmlHttpRequest` is easier to read than `XMLHTTPRequest` (and both would be better than the actual insane `XHLHttpRequest`)

Reply:
I guess HogQL is kind of a special case, because it's part acronym and part word